tests/integration/test_charm.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+
+import pytest
+
+log = logging.getLogger(__name__)
+
+
+async def file_contents(unit, path):
+    cmd = "cat {}".format(path)
+    action = await unit.run(cmd)
+    return action.results["Stdout"]
+
+
+@pytest.mark.abort_on_fail
+async def test_build_and_deploy(ops_test):
```

Comment:
Usually this is a fixture like [here](https://github.com/canonical/flask-k8s-operator/blob/main/tests/integration/conftest.py#L95) in Flask.

Reply:
This is in the progress of being change as a part of add integration tests.